src/contract_wrappers/registries/security_token_registry_wrapper.ts

```diff
@@ -658,11 +675,12 @@ export default class SecurityTokenRegistryWrapper extends ContractWrapper {
    * Sets the ticker registration fee in POLY tokens. Only Polymath.
    */
   public changeTickerRegistrationFee = async (params: ChangeFeeParams) => {
+    const newFeeWei = valueToWei(params.newFee, FULL_DECIMALS);
     await this.checkOnlyOwner();
     const actualFee = await this.getTickerRegistrationFee();
-    assert.assert(!actualFee.eq(params.newFee), 'Fee not changed');
+    assert.assert(!actualFee.eq(newFeeWei), 'Fee not changed');
```

Comment:
actual fee is not in wei, newFeeWei is in wei. This comparison is wrong. You should convert this value after this comparison.